packages/web3-eth-txpool/package.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "web3-eth-txpool",
+    "namespace": "ethereum",
+    "version": "1.0.0-beta.52",
+    "description": "Web3 module to interact with the Ethereum blockchain accounts stored in the node.",
+    "repository": "https://github.com/ethereum/web3.js/tree/1.0/packages/web3-eth-txpool",
+    "license": "LGPL-3.0",
+    "main": "dist/web3-eth-txpool.cjs.js",
+    "module": "dist/web3-eth-txpool.esm.js",
+    "browser": "dist/web3-eth-txpool.umd.js",
+    "scripts": {
+        "build": "rollup -c",
+        "dev": "rollup -c -w",
+        "test": "jest",
+        "dtslint": "dtslint types --onlyTestTsNext"
+    },
+    "types": "types/index.d.ts",
+    "dependencies": {
```

Comment:
I will check and fix the dependencies of the new modules after I've merged all of them.

Reply:
Ya cool.